doc/_static/style.css

```diff
@@ -131,3 +136,34 @@ a.copybtn {
     user-select: none;
 }
 
+/* Atkinson Hyperlegible regular */
+@font-face {
+    font-family: "Atkinson Hyperlegible";
+    src: url("./fonts/Atkinson-Hyperlegible-Regular-102a.woff2") format('woff2');
```

Comment:
Just a note on the Web Open Font Format 2 (woff2) format, it's smaller to serve than woff, but not supported on Intetnet Explorer (see https://caniuse.com/woff2). Can we check our Google Analytics added in #314 to see how many users this equates to? Just don't want to accidentally affect a significant proportion of our users.

Reply:
https://en.wikipedia.org/wiki/Internet_Explorer#End_of_life

> Microsoft Teams no longer support IE11 as of November 30, 2020, and the remaining O365 applications will no longer support IE11 on August 17, 2021.

We don't need to worry about IE.

Reply:
Cool, and if there is anyone using IE, they will still have a 'pretty' default sans-serif font to look at.